src/test/java/org/broadinstitute/dsde/consent/ontology/datause/services/TextTranslationServiceImplTest.java

```diff
@@ -89,6 +112,19 @@ public void testTranslateDataset() throws IOException {
         assertTrue(translation.contains("[GRU]"));
     }
 
+    @Test
+    public void testTranslateParagraph() throws Exception {
+      HttpResponse response = loadTermItemsResponse();
+      when(gcsStore.getStorageDocument(Mockito.anyString())).thenReturn(response);
+      String mockParagraph = "GRU General research for some test with disease. This is not for profit.";
+
+      HashMap<String, Recommendation> translation = service.translateParagraph(mockParagraph);
+      assertEquals(3, translation.size());
+      assertEquals("General Research Use", translation.get("http://purl.obolibrary.org/obo/DUO_0000042").title());
+      assertEquals("Disease Specific Research", translation.get("http://purl.obolibrary.org/obo/DUO_0000007").title());
+      assertEquals("Not for Profit Organization Use Only", translation.get("http://purl.obolibrary.org/obo/DUO_0000045").title());
+    }
```

Comment:
Great test. I think we could also use a service level test for some interesting cases:

- "Science"
- ""
- "for profit"
- "FOR PROFIT"

This is also a great fit for parameterized tests: https://www.baeldung.com/parameterized-tests-junit-5